### PR TITLE
CU_86bznb0a7: Fix/omang env var

### PR DESCRIPTION
**Pull request by ItsMurumba**
## Type of PR (check all applicable)

- [ ] 💼 New Package
- [ ] ♻️ Refactor
- [ ] ✨ New Feature
- [x] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🧪 Tests
- [ ] 🔖 Release
- [ ] 🚩 Other

## Description

This PR [adds/removes/fixes/replaces] this [feature/bug/etc]. 

## Tickets & Related Documents

Resolves #

## Added code snippets?
- [ ] 👍 Yes
- [x] 🙅 No, because they aren't needed

## Added tests?

- [ ] 👍 Yes
- [x] 🙅 No, because they aren't needed
- [ ] 🙋 No, because I need help

## Added to documentation?

- [ ] 📜 readme
- [ ] 📓 general documentation
- [ ] 🙅 no documentation needed

## Are there any post-deployment tasks we need to perform?

-
-
-

### CODE REVIEW

**Review by marrouchi — CHANGES_REQUESTED** [review]
```
[3:51:31 PM] Starting compilation in watch mode...
2024-07-25 18:51:31 
2024-07-25 18:51:34 [3:51:34 PM] Found 0 errors. Watching for file changes.
2024-07-25 18:51:34 
2024-07-25 18:51:35 [Nest] 29  - 07/25/2024, 3:51:35 PM     LOG [NestFactory] Starting Nest application...
2024-07-25 18:51:35 [Nest] 29  - 07/25/2024, 3:51:35 PM     LOG [InstanceLoader] UserModule dependencies initialized +50ms
2024-07-25 18:51:35 [Nest] 29  - 07/25/2024, 3:51:35 PM     LOG [InstanceLoader] TypeOrmModule dependencies initialized +0ms
2024-07-25 18:51:35 [Nest] 29  - 07/25/2024, 3:51:35 PM     LOG [InstanceLoader] HttpModule dependencies initialized +1ms
2024-07-25 18:51:35 [Nest] 29  - 07/25/2024, 3:51:35 PM   ERROR [ExceptionHandler] NJS-101: no credentials specified
2024-07-25 18:51:35 Error: NJS-101: no credentials specified
2024-07-25 18:51:35     at Object.throwErr (/app/node_modules/oracledb/lib/errors.js:636:10)
2024-07-25 18:51:35     at Object.checkCredentials (/app/node_modules/oracledb/lib/thin/util.js:236:14)
2024-07-25 18:51:35     at ThinPoolImpl._init (/app/node_modules/oracledb/lib/thin/pool.js:47:14)
2024-07-25 18:51:35     at ThinPoolImpl.create (/app/node_modules/oracledb/lib/thin/pool.js:121:10)
2024-07-25 18:51:35     at Object.createPool (/app/node_modules/oracledb/lib/oracledb.js:603:22)
2024-07-25 18:51:35     at processTicksAndRejections (node:internal/process/task_queues:95:5)
```
Omang API is failing to connect to the Oracle DB, there seem to be an issue with the supplied credentials.

File: .env.local [hunk 1]
```diff
@@ -30,8 +30,8 @@ KAFKA_TOPICS=send-adt-to-ipms,send-orm-to-ipms,save-pims-patient,save-ipms-patie
 OMANG_DEV_MOUNT_FOLDER=
 # For MacOs : build locally and use OMANG_ORACLE_DOCKER_IMAGE=oracle/database:19.3.0-ee-slim-faststart
 # For Linux : build locally and use OMANG_ORACLE_DOCKER_IMAGE=gvenzl/oracle-free   
-# OMANG_ORACLE_DOCKER_IMAGE=
-# OMANG_ORACLE_SID=FREE
+OMANG_ORACLE_DOCKER_IMAGE=
+OMANG_ORACLE_SID=FREE
```
Comment (marrouchi) [inline .env.local:34]: Please roll back this change

File: packages/omang-service-mediator/docker-compose.dev.yml [hunk 2]
```diff
@@ -4,7 +4,7 @@ services:
   oracle-db:
     image: ${OMANG_ORACLE_DOCKER_IMAGE}
     environment:
-      - ORACLE_SID=${OMANG_ORACLE_SID}
+      - ORACLE_SID=${OMANG_ORACLE_DB_SID}
```
Comment (marrouchi) [inline packages/omang-service-mediator/docker-compose.dev.yml:7]: in the env file the variable name is OMANG_ORACLE_SID

File: projects/omang-service-mediator/package.json [hunk 3]
```diff
@@ -35,6 +35,7 @@
     "@smile-cdr/fhirts": "^2.1.1",
     "axios": "^1.6.8",
     "cookie-parser": "^1.4.6",
+    "dotenv": "^16.4.5",
```
Comment (marrouchi) [inline projects/omang-service-mediator/package.json:38]: Do we still use this ?

**Review by marrouchi — CHANGES_REQUESTED** [review]

File: packages/omang-service-mediator/docker-compose.dev.yml (hunk 2 above)
Comment (ItsMurumba) [reply packages/omang-service-mediator/docker-compose.dev.yml:7]: I have updated the variable as well. I made changes just to standardize the variables naming

File: projects/omang-service-mediator/package.json (hunk 3 above)
Comment (ItsMurumba) [reply projects/omang-service-mediator/package.json:38]: No we don't. Let me clean that up.

**Review by marrouchi — CHANGES_REQUESTED** [review]

**Pull request merged**